client/updateCenter.go

```diff
@@ -153,3 +159,37 @@ func (u *UpdateCenterManager) DownloadJenkins(lts bool, output string) (err erro
 	err = downloader.DownloadFile()
 	return
 }
+
+func (u *UpdateCenterManager) GetSite() (site *CenterSite, err error) {
+	req := u.commonGet("/updateCenter/site/default/api/json?pretty=true&depth=2")
```

Comment:
Please use the new method (`RequestWithData`) instead of this style.

See also https://github.com/jenkins-zh/jenkins-cli/blob/master/client/pluginManger.go#L94